### DISCUSSION TITLE
Confirm the removal of a website

### DISCUSSION THREAD

**Discussion by michsacco** [umami-software/umami#2016] [Q&A] [answered]
Good Morning
i've been using umami (self-hosted) since a few week and now i'm going through an issue:
i have to delete a website from my dashboard and their relative data. In the UI, a popup ask me the confirmation; i've been trying with the name of the zone, with the website url, with the website ID and last with the account password... but nothing.
In the docs i don't find the relative section or a tutorial.
Please help me with this misunderstanding 😅
Thanks.

**Comment by mikecao** [comment, marked as answer]
You need to type `DELETE` to remove a website.

**Reply by michsacco** [reply]
Thank you so much

**Reply by FHU-yezi** [reply]
It is missing in some translations, such as Chinese, maybe due to translate mistake? Can we fix it?
